### DISCUSSION TITLE
Global labels

### DISCUSSION THREAD

**Discussion by glewe** [community/community#10374] [Projects and Issues]
Hi there,

I am not so happy with the default labels for repo issues. I usually change them a bit to fit my needs. However, I have to do the same changes for each of my repos. I would wish for a global label configuration for my account so that all repos get them.

Would that be something worth a thought?

Best regards,
George

**Comment by TwiN** [comment]
For what it's worth, GitHub organizations has a tab named `Repository defaults` in which default repository labels can be configured.

I wish this feature was also available for normal users too, as I'm starting to have a lot of repositories and have to consider using GitHub actions to synchronize labels across all my repositories instead.

**Comment by mcardle** [comment]
This features would be amazing especially for Github Projects.

**Comment by ghost** [comment]
This would be a very welcome feature. Default labels for organizations are not enough, because if you already have a lot of repositories having been created before defining the default labels, you're out of luck.

There should be "global labels" for an organization, so I can reuse them between repositories Issues, and inside projects without duplicating them in several repositories.

**Comment by jacobwatkinsgit** [comment]
+1, except I'd rather a means to do it per-repository. A file, or automated way, to easily import/export labels and colors. Preferably hidden away in the .git folder.

**Comment by iPoetDev** [comment]
I am doing a front-end web client/server/api that attempts to solve this by being a 3rd party control plane for all repositories and over an api (college assignment)
